docs/advanced_topics.rst

```diff
@@ -0,0 +1,43 @@
+Advanced topics
+=================
+
+In this section, you can find some advanced functionalities and topics that are just partially mentioned in the API description. For the typical use case, DALI default configuration should perform well out of the box and you don't need to worry about those.
+
```

Comment:
This section covers a few advanced topics that are mentioned in the API description. However, for the typical use case, DALI default configuration will perform well out of the box, and you don't need to worry about those.

Reply:
Adjusted